web/src/locales/en.json

```diff
@@ -256,6 +256,7 @@
       "disable-public-memos": "Disable public memos",
       "max-upload-size": "Maximum upload size (MiB)",
       "max-upload-size-hint": "Recommended value is 32 MiB.",
+      "vacuum-hint": "Cleans up unused or obsolete data.",
```

Comment:
I think obsolete is (nearly) a synonym of unused. I personally think only 'unused' would look better.

Reply:
Ok, thanks. Appreciate it